### PR TITLE
Write a blog post for the upcoming DBA changes

### PR DESCRIPTION
**Pull request by imprashast**
Wait for all releases to go thru before merging this

### CODE REVIEW

File: docs/blog/posts/2024/warp-2-0.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+### How to Update Your Apps for DBA
+
+#### Packages to Install from NPM
+
+| Package Name       | Version        |
+| ------------------ | -------------- |
+| `@warp-ds/css`     | `2.0.0-next.1` |
+| `@warp-ds/drive`   | `2.0.0`        |
+| `@warp-ds/react`   | `2.0.0-next.0` |
+| `@warp-ds/vue`     | `2.0.0-next.0` |
```
Comment (felicia-haggqvist) [inline docs/blog/posts/2024/warp-2-0.md]: I'm assuming that @warp-ds/vue will get `3.0.0-next.1` and not `2.0.0-next.0`, once we include the breaking changes and make a new release to next? :)  The current version is at `2.0.1-next.4`

File: docs/blog/posts/2024/warp-2-0.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+### How to Update Your Apps for DBA
+
+#### Packages to Install from NPM
+
+| Package Name       | Version        |
+| ------------------ | -------------- |
+| `@warp-ds/css`     | `2.0.0-next.1` |
+| `@warp-ds/drive`   | `2.0.0`        |
+| `@warp-ds/react`   | `2.0.0-next.0` |
```
Comment (felicia-haggqvist) [inline docs/blog/posts/2024/warp-2-0.md]:
```suggestion
| `@warp-ds/react`   | `2.0.0-next.1` |
```

File: docs/blog/posts/2024/warp-2-0.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+| `@warp-ds/vue`     | `2.0.0-next.0` |
+| `@warp-ds/elements`| `2.0.0-next.0` |
```
Comment (felicia-haggqvist) [inline docs/blog/posts/2024/warp-2-0.md]:
```suggestion
| `@warp-ds/elements`| `2.0.0-next.1` |
```

File: docs/blog/posts/2024/warp-2-0.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+---
+
+### How to Update Your Apps for DBA
+
+#### Packages to Install from NPM
+
+| Package Name       | Version        |
+| ------------------ | -------------- |
+| `@warp-ds/css`     | `2.0.0-next.1` |
```
Comment (felicia-haggqvist) [inline docs/blog/posts/2024/warp-2-0.md]: `2.0.0-next.1` is the one we have pointed v2 alias to, but now the current version on next is at `2.0.0-next.2` and once we make a stable release of drive (@warp-ds/uno), there will be a new PR in the css-repo that would then bump next-version to `2.0.0-next.3`. So, my question is, do we want it to say here `2.0.0-next.1` or `2.0.0-next.3`? :)

File: docs/blog/posts/2024/warp-2-0.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+| `@warp-ds/drive`   | `2.0.0`        |
```
Comment (felicia-haggqvist) [inline docs/blog/posts/2024/warp-2-0.md]:
```suggestion
| `@warp-ds/uno`   | `2.0.0`        |
```

File: docs/blog/posts/2024/warp-2-0.md (hunk 4 above)
Comment (imprashast) [reply docs/blog/posts/2024/warp-2-0.md]: Yeah next.3 makes sense. Updated

File: docs/blog/posts/2024/warp-2-0.md (hunk 1 above)
Comment (imprashast) [reply docs/blog/posts/2024/warp-2-0.md]: You're right. Fixed

**Review by felicia-haggqvist — APPROVED** [review]

**Pull request merged**